docs/src/main/sphinx/admin/properties.rst

```diff
@@ -21,3 +21,80 @@ may be used to tune Trino or alter its behavior when required.
     Logging <properties-logging>
     Web UI <properties-web-interface>
     Regular expression function <properties-regexp-function>
+
+Property value types
+--------------------
+
+Configuration properties in Trino take values of different types, some standard
+and some non-standard. Supported values may be further limited on a per-property
+basis.
```

Comment:
Are we talking about Duration as being non-standard? If so that's true (only because the actual standard for durations is not human friendly - https://www.digi.com/resources/documentation/digidocs/90001437-13/reference/r_iso_8601_duration_format.htm) but I don't think this language contributes anything meaningful to the reader.